WordPress/src/main/res/layout/notifications_fragment_notes_list.xml

```diff
@@ -205,41 +210,52 @@
 
     </LinearLayout>
 
-
-    <org.wordpress.android.widgets.WPNestedScrollView
+    <LinearLayout
         android:id="@+id/connect_jetpack"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:visibility="gone">
+        android:gravity="center"
+        android:orientation="vertical">
 
-        <LinearLayout
+        <ScrollView
```

Comment:
I think `NestedScrollView` should be used within `CoordinatorLayout` cause `ScrollView` doesn't work well.

Reply:
I updated the view in 8f743ba.